src/views/HomeView.vue

```diff
@@ -21,48 +18,23 @@
             <div id="game-list">
               <p v-if="gameList.length === 0" data-cy="text-if-no-game">No Active Games</p>
               <div v-for="game in gameList" :key="game.id">
-                <game-list-item
-                  :name="game.name"
-                  :p0ready="game.p0Ready ? 1 : 0"
-                  :p1ready="game.p1Ready ? 1 : 0"
-                  :game-id="game.id"
-                  :status="game.status"
-                  :num-players="game.numPlayers"
-                  :is-ranked="game.isRanked"
-                />
+                <game-list-item :name="game.name" :p0ready="game.p0Ready ? 1 : 0" :p1ready="game.p1Ready ? 1 : 0"
+                  :game-id="game.id" :status="game.status" :num-players="game.numPlayers" :is-ranked="game.isRanked" />
```

Comment:
We generally prefer one-line-per attribute for template code when the tag has many attributes. Please roll this back to the previous multi-line setup. I assume that this was done automatically by the linter/IDE? We clearly could use an improvement to the lint setup to make this more consistent. I think that warrants a separate issue, but for now, please keep the multiple-lines for these longer tags.

Reply:
Got it.